experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -26,5 +27,72 @@ function atmos_init(::Type{FT}, Y, integrator; params = nothing) where {FT}
         @warn("Running with ρe_int in coupled mode is not tested yet.")
     end
 
-    AtmosSimulation(params, Y, spaces, integrator)
+    ClimaAtmosSimulation(params, Y, spaces, integrator)
+end
+
+# required by the Interfacer
+get_field(sim::ClimaAtmosSimulation, ::Val{:radiative_energy_flux}) = level(sim.integrator.p.ᶠradiation_flux, half)
+get_field(sim::ClimaAtmosSimulation, ::Val{:liquid_precipitation}) =
+    sim.integrator.p.col_integrated_rain .+ sim.integrator.p.col_integrated_snow # all fallen snow melts for now
+get_field(sim::ClimaAtmosSimulation, ::Val{:snow_precipitation}) = sim.integrator.p.col_integrated_snow .* FT(0)
```

Comment:
I know it was not introduced in this PR, but we can extract `FT` as 
```
FT = Spaces.undertype(axes(Y.c.ρe_tot))
```
(or any other `Y` field) instead of passing it as an argument to the `atmos_init`